pkg/sql/copy/copy_test: TestCopyInReleasesLeases failed

pkg/sql/copy/copy_test.TestCopyInReleasesLeases [failed](https://mesolite.cluster.engflow.com/invocations/default/86087bb5-1c9e-41b9-881c-fab9c668231c?testReportRun=1&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvc3FsL2NvcHk6Y29weV90ZXN0) on release-25.3 @ [3bc58d4160d543f0721761e921cf5f9887bf55bc](https://github.com/cockroachdb/cockroach/commits/3bc58d4160d543f0721761e921cf5f9887bf55bc):


```
=== RUN   TestCopyInReleasesLeases
    test_log_scope.go:165: test logs captured to: outputs.zip/logTestCopyInReleasesLeases1512015655
    test_log_scope.go:76: use -show-logs to present logs inline
    test_server_shim.go:156: automatically injected an external process virtual cluster under test; see comment at top of test_server_shim.go for details.
    copy_in_test.go:783: alter did not complete
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestCopyInReleasesLeases1512015655
--- FAIL: TestCopyInReleasesLeases (103.89s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>race=true</code>
 - <code>run=1</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestCopyInReleasesLeases.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-55384

--- a/pkg/sql/copy/copy_in_test.go
+++ b/pkg/sql/copy/copy_in_test.go
@@ -779,7 +779,7 @@ func TestCopyInReleasesLeases(t *testing.T) {
 	select {
 	case err := <-alterErr:
 		require.NoError(t, err)
-	case <-time.After(testutils.DefaultSucceedsSoonDuration):
+	case <-time.After(testutils.SucceedsSoonDuration()):
 		t.Fatal("alter did not complete")
 	}
 	require.NoError(t, conn.Close(ctx))

